src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -213,7 +213,7 @@ static void wrapped_rr_closure(grpc_exec_ctx* exec_ctx, void* arg,
       grpc_grpclb_client_stats_unref(wc_arg->client_stats);
     }
     if (GRPC_TRACER_ON(grpc_lb_glb_trace)) {
-      gpr_log(GPR_INFO, "Unreffing RR %p", (void*)wc_arg->rr_policy);
+      gpr_log(GPR_INFO, "Unreffing RR %p", wc_arg->rr_policy);
```

Comment:
Should we include the address of the grpclb policy in this log message?

Reply:
Same as in the other comment, `glb_policy` not available. Possible to make it, but it'd only be used for logging. Still, it may be worthwhile, given how much easier it is to debug issues with good logging.